packages/core/src/messagingProtocols/mls/E2EIdentityService/E2EIServiceInternal.ts

```diff
@@ -102,12 +102,9 @@ export class E2EIServiceInternal {
 
   // ############ Internal Functions ############
 
-  private async initIdentity(hasActiveCertificate: boolean) {
+  private async initIdentity(hasActiveCertificate: boolean, ciphersuite: Ciphersuite) {
     const {user} = this.initialData;
 
-    // How long the issued certificate should be maximal valid
-    const ciphersuite = Ciphersuite.MLS_128_DHKEMX25519_AES128GCM_SHA256_Ed25519;
-
```

Comment:
by "missed", you mean it wasn't removed and we should get it from the config only?

Reply:
Exactly :)